#56 :bug: allow local monorepo dependencies to be excluded from requi… (#58)
TechnologyBrewery/habushu: README.md

Habushu will replace invocations of Poetry's `build` and `publish` commands with the extensions of those commands exposed by the
`poetry-monorepo-dependency-plugin`, which are `build-rewrite-path-deps` and `publish-rewrite-path-deps`, respectively.

To further ease working with monorepo dependencies, Habushu will specially handle how these dependencies are written to 

> intermediate `requirements.xml` exports if they are included in the `[tool.poetry.group.monorepo.dependencies]` group 

Reviewer: A:
```suggestion
intermediate `requirements.txt` exports if they are included in the `[tool.poetry.group.monorepo.dependencies]` group 
```

Author: ✅ Good catch - fixed!

> with your `pyproject.toml` file.  In this case, dependencies will excluded from the export unless `rewriteLocalPathDepsInArchives`

Reviewer: A:
```suggestion
within your `pyproject.toml` file.  In this case, the dependencies will be excluded from the export unless `rewriteLocalPathDepsInArchives`
```

Author: ✅ good catch - fixed!

is set to true.  This feature can be useful if installing the exported `requirements.txt` on another machine and/or 

> Docker container.  If dependencies are not included in this specific group, the dependencies will exported with local 

Reviewer: A:
```suggestion
Docker container.  If a monorepo dependency is not included in this specific group, it will exported to with a local 
```

Author: ✅ Good comment - fixed.
